src/main/kotlin/com/github/rushyverse/pvpbox/listener/PlayerSpawnListener.kt

```diff
@@ -33,6 +34,8 @@ class PlayerSpawnListener(
 
         player.teleport(spawnPoint)
 
+        MapImage.packets()?.toList()?.let { player.sendPackets(it) }
+
```

Comment:
Use instance and not singleton (more simple for tests)

Reply:
Done here https://github.com/Rushyverse/pvpbox/commit/e993f5cf0c24d147648252dc0962ee85cf429995#diff-f7e024fbf9acbca3b1e549e70f435c074d786e7499ad81b566fda754c4759b3f